src/workflow/StreamlitUI.py

```
@@ -480,54 +534,74 @@ def format_files(input: Any) -> List[str]:
                     name=name,
+                    help=help,
+                    widget_type="number",
+                    options=None,
                     min_value=min_value,
                     max_value=max_value,
                     step_size=step_size,
-                    help=help,
+                    display_file_path=False,
+                    on_change=on_change,
                 )
             elif (isinstance(value, str) or value == None) and options is not None:
```

Ruff (0.15.4)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)